src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -264,13 +285,29 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     })()
   }, [accountId, assetId, chainId, fromAddress])
 
+  const memo = useMemo(() => {
+    if (isRunePool && state) {
+      const balanceCryptoPrecision = bnOrZero(amountAvailableCryptoPrecision.toPrecision())
+      const percent = bnOrZero(state.withdraw.cryptoAmount)
+        .div(balanceCryptoPrecision)
+        .times(100)
+        .toFixed(0)
+      const basisPoints = convertPercentageToBasisPoints(percent)
+
+      return `pool-:${basisPoints}`
+    }
+    if (quote?.memo) return quote.memo
+
+    return null
+  }, [isRunePool, quote, state, amountAvailableCryptoPrecision])
+
   const { executeTransaction, estimatedFeesData } = useSendThorTx({
     accountId: accountId ?? null,
     assetId,
-    // withdraw savers will use dust amount
+    // withdraw savers will use dust amount if withdrawSavers, 0 if withdrawRunepool
```

Comment:
suggestion: revert me to avoid confusion, since we now have savers/runepool terminology across the board, the current comment implies this only applies to savers and that runepool *won't* use dust amount

Reply:
Reverted in [1f9d825](https://github.com/shapeshift/web/pull/7449/commits/1f9d8254294f6843bde971347039e344909ac032)